spec/__qnaTests__/AnswerQuestion.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import 'regenerator-runtime/runtime';
+import { test, expect, afterEach } from '@jest/globals';
+import {
+  render, screen, cleanup, fireEvent,
```

Comment:
I think we should possibly be using user-event instead of fireEvent to test our events here.

https://testing-library.com/docs/user-event/intro/#difference-to-fireevent